science/nds2-client/Portfile

```diff
@@ -13,117 +16,169 @@ long_description \
   Client tool for accessing streamed LIGO data using the Network \
   Data Server version 2.
 
-homepage          https://www.lsc-group.phys.uwm.edu/daswg/projects/nds-client.html
-master_sites      https://www.lsc-group.phys.uwm.edu/daswg/download/software/source/
+homepage          https://wiki.ligo.org/DASWG/NDSClient
+master_sites      http://software.ligo.org/lscsoft/source/
 
-checksums         rmd160  ca470079f3d908cb95b379d257374e3b91863034 \
-                  sha256  3abc8ec2ca58fc3184e46b14b81b1b87882b1abe4e739c7c0ae35e1ad77b5f20
+checksums         rmd160  b8b00f8de773d8149b0552b9369bde3c09dc6768 \
+                  sha256  a3cca2d5490cf22428120d4b5c7fbe8c5fd0dcc9c1abe6a64eef68c5029ab63e
 
 depends_build-append \
-                  port:pkgconfig
+                  port:pkgconfig \
+				  port:swig
```

Comment:
My latest commit has changed tabs to spaces.

As for squashing, I would like to but I am unsure how. Is there a command line procedure or web interface procedure?

My impression of timeout is from https://travis-ci.org/macports/macports-ports/jobs/252469898

This is seen for all three systems and the result in the log you showed could be the result of the system terminating the process?